src/components/layout/web/Seo.js

```diff
@@ -9,9 +10,9 @@ export default function Seo(props) {
     title: `${props.title || `Impact sur le climat des objets et gestes`} | Impact CO2`,
     description:
       props.description ||
-      `Découvrez l’impact sur le climat des objets et gestes de votre quotidien comme votre mobilier, vos habits ou encore vos repas`,
-    image: `https://${process.env.websiteurl || 'impactco2.fr'}/${props.image || 'metaimage.png'}`,
-    url: `https://${process.env.websiteurl || 'impactco2.fr'}${router.asPath}`,
+      `Sensibilisez votre communauté grâce aux ressources sur l’impact carbone des gestes et objets du quotidien,  issue des données environnementales de l'ADEME.`,
```

Comment:
pas besoin d'un interpreteur de string ici
```suggestion
      'Sensibilisez votre communauté grâce aux ressources sur l’impact carbone des gestes et objets du quotidien,  issue des données environnementales de l'ADEME.',
```